tests/callbacks/test_activation_monitor.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from torch.utils.data import DataLoader
+
+from composer.callbacks import ActivationMonitor
+from composer.loggers import InMemoryLogger
+from composer.optim import DecoupledAdamW
+from composer.trainer import Trainer
+from tests.common.datasets import RandomClassificationDataset
+from tests.common.models import SimpleModel
+
+
+def test_activation_monitor():
```

Comment:
I'd like to test some of your helper functions as well, such as compute_kurtosis and add_metrics

Reply:
What tests would you like to see for `add_metrics`? Since what it fundamentally does is compute statistics over `d_model` with torch functions